packages/orama/src/components/algorithms.ts

```diff
@@ -16,16 +17,24 @@ export function prioritizeTokenScores(arrays: TokenScore[][], boost: number, thr
     for (let j = 0; j < entriesLength; j++) {
       const [token, score] = arr[j]
       const boostScore = score * boost
+      const oldScore = tokenMap.get(token)
```

Comment:
If I remember correctly, get has very bad performance when the key is not found. It is better to use `has` and then `get`